sdk/storage/azure-storage-file-share/src/main/java/com/azure/storage/file/share/ShareClientBuilder.java

```diff
@@ -10,12 +10,13 @@
 import com.azure.core.client.traits.ConnectionStringTrait;
 import com.azure.core.client.traits.EndpointTrait;
 import com.azure.core.client.traits.HttpTrait;
+import com.azure.core.client.traits.TokenCredentialTrait;
```

Comment:
What about the DirectoryClientBuilder and LeaseClientBuilder?

Reply:
DirectoryClientBuilder does not exist, directoryClient can be created from the ShareFileClientBuilder (which I've added the new property to). And for LeaseClientBuilder, I've already added the new properties to ShareLeaseClientBuilder (under the specialized folder).